src/components/ebay-alert-dialog/index.marko

```diff
@@ -4,24 +4,23 @@ $ var mainId = component.getElId("alert-dialog-main");
     ...input
     role="alertdialog"
     open=input.open
-    classPrefix="lightbox-dialog"
+    classPrefix="alert-dialog"
     ignore-escape=true
-    window-type="compact"
     main-id=mainId
     on-open("emit", "open")
     on-close("emit", "close")
     buttonPosition="hidden"
     focus=confirmId
-    class=[input.class, "lightbox-dialog--mask-fade"]
-    window-class=["lightbox-dialog__compact-window--fade"]>
+    class=[input.class, "alert-dialog--mask-fade"]
+    window-class=["alert-dialog__window alert-dialog__window--fade"]>
     <@footer>
         <ebay-button
             priority="primary"
             aria-describedby=mainId
             on-click("emit", "confirm")
             key="confirm"
             id=confirmId
-            class="lightbox-dialog__confirm">
+            class="alert-dialog__confirm">
```

Comment:
I think the class here was alert-dialog__acknowledge -> Ian changed it in his last PR

Reply:
I don't see the class being set in css/less. I only see it shown in the docs. So does it matter since the class isn't really being used?

Reply:
You're right it probably doesn't matter then. If that's the case we should drop it from the skin docs to avoid confusion

Reply:
Just updated it in ebayui for consistency